locationsyncservice/src/main/java/com/shuttl/location_pings/config/open_lib/LocationsHelper.kt

```diff
@@ -33,14 +33,17 @@ object LocationsHelper {
         LocationRetrofit.networkDebug = inteceptor
     }
 
-    fun initLocationsModule(app: Application, interceptor: Interceptor? = null, locationConfigs: LocationConfigs) {
+    fun initLocationsModule(
+        app: Application,
+        interceptor: Interceptor? = null,
+        locationConfigs: LocationConfigs
+    ) {
         setNetworkingDebug(interceptor)
         val pingIntent = Intent(app, LocationPingService::class.java)
         pingIntent.putExtra("config", locationConfigs)
         val saveIntent = Intent(app, LocationSaveService::class.java)
         saveIntent.putExtra("config", locationConfigs)
         app.startService(pingIntent)
-        app.startService(pingIntent)
```

Comment:
add: app.startService(saveIntent)
remove: app.bindService(...

Reply:
Removed : bindService() and serviceConnection() method
Added : app.startService(saveIntent)